tools/benchmark/generate/main.go

```diff
@@ -723,7 +723,7 @@ func main() {
 		}
 		log.Printf("Generating %s...", b.Filename)
 		todo <- nil
-		go func() {
+		go func(b *RebuildBenchmark) {
```

Comment:
ugh. this is no longer necessary when we raise the minimum supported go version but it's fine for now.